addons/l10n_ec/models/l10n_ec_sri_payment.py

```diff
@@ -8,6 +8,9 @@ class SriPayment(models.Model):
 
     _name = "l10n_ec.sri.payment"
     _description = "SRI Payment Method"
+    _order = 'active DESC, sequence, id'
 
-    name = fields.Char("Name")
+    sequence = fields.Integer(default=10)
```

Comment:
@jco-odoo @odoostan there are 2 new fields that we won't be able to include in stable, but yes for master (V16), and those 2 are here.
cc' @ren-odoo @fvz-odoo

Reply:
Which are the two fields exactly ?

Reply:
Should be included in l10n_ec_edi instead idd.

Reply:
@vbe-odoo this comment is ready and should just be closed, would you close it, please? the extra fields were removed as explained in my last commit.